src/components/Trackpad/ControlBar.tsx

```
@@ -2,153 +2,100 @@ import { ModifierState } from "@/types";
+				return "Release";
+			case "Release":
+				return "Hold";
+		}
+	};
+
+	return (
+		<div className="bg-base-200 p-2 grid grid-cols-5 gap-2 shrink-0">
+			<button
+				className={`btn btn-sm ${scrollMode ? "btn-primary" : "btn-outline"}`}
+				onPointerDown={(e) => handleInteraction(e, onToggleScroll)}
+			>
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)